spec/interactors/users/check_credentials_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+describe Users::CheckCredentials do # rubocop:disable Metrics/BlockLength
+  describe ".call" do # rubocop:disable Metrics/BlockLength
+    let!(:user) do
+      {
+        first_name: "test",
+        last_name: "test",
+        email: "person@admin.ru",
+        password: "123456"
+      }
+    end
+
+    context "when credentials are valid" do
+      let!(:spec_credentials) do
+        {
+          first_name: "test",
+          last_name: "test",
+          email: "person@admin.ru",
+          password: "123456"
+        }
+      end
+
+      let!(:interactor) do
+        described_class
+          .new(credentials: spec_credentials)
+      end
```

Comment:
Why it is multiline?

Reply:
it decision was accepted after rubocop

Reply:
i don't understanding why this work in one-line rn :)

Reply:
Rewrite this spec. let(:interactor) should be in top of file